containers/src/Kokkos_ScatterView.hpp

```diff
@@ -965,10 +970,20 @@ class ScatterView<DataType, Kokkos::LayoutRight, DeviceType, Op,
 
   template <typename RT, typename... RP>
   ScatterView(View<RT, RP...> const& original_view)
+      : ScatterView(view_alloc(execution_space()), original_view) {}
+
+  template <typename RT, typename... P, typename... RP>
+  ScatterView(::Kokkos::Impl::ViewCtorProp<P...> const& arg_prop,
```

Comment:
Consistency with how it was done in the earlier PR, which I believe was at the direction to do it how it's done elsewhere (e.g. `View` itself)

Reply:
I actually think this constructor is somewhat different.  The motivation for using  `ViewCtorProp` at other places was consistency with `View` constructors but that constructor has no equivalent in `View`.
Besides one might be surprised that the implementation ignore the label if present (I am not suggesting we should).  At the end of the day, if all you want is an execution space, I would not bother with wrapping it.
Let's see what others think.

Reply:
So, you actually added the initial constructors that take `ViewCtorProp`, since I really don't have a great handle on what's the Kokkos way here. If you think just passing the execution space is preferable, I'll just do that.

Reply:
Yeah, I prefer passing the ExecSpace directly. I'm not sure about whether it's idiomatic, as I'm not really sure about the purpose of ViewCtorProp's, but certainly just passing the space reads better